src/styles/block/pruduct-card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__image {
+    justify-content: center;
+    display: flex;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__image-content {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin-top: 40px;
+    margin-bottom: 0;
+    padding: 0;
+    color: $primary-color;
+    font-weight: 500;
```

Comment:
The font-size should be 11px according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Please correct it to match the guideline.